node/resolvers/sponsoredBanners.ts

```diff
@@ -0,0 +1,51 @@
+import AdServer from '../clients/AdServer'
+import type { AdServerSponsoredBannersResponse } from '../typings/AdServer'
+
+const AD_TYPE = 'banner'
+const SPONSORED_COUNT = 1
+
+const mapSponsoredBanner = (
```

Comment:
I think `mapSponsoredBanner` doesn't do justice to what the function does. You could "map" a function for many reasons. Even though it describes the method you used, it doesn't describe what the function in fact does.